.circleci/config.yml

```diff
@@ -54,6 +54,9 @@ jobs:
       - run:
           name: install-npm-wee
           command: npm install
+      - run:
+          name: install-npm-inner
+          command: cd auto_instrument_handler; npm install
```

Comment:
I think its better to use pushd and popd, after this the circle while be on `auto_instrument_handler` folder